utils/cLog.cpp

```diff
@@ -1,262 +1,204 @@
-#include "../include/d2tmh.h"
 #include "cLog.h"
 
+#include "definitions.h"
+
+#include <cstdio>
+#include <ctime>
+#include <system_error>
+
+extern bool	bDoDebug;
```

Comment:
I put this one here, instead of including the entire header file with all the globals.
It's not ideal, but for now, I think this is a step forward.